### PR TITLE
Upload multiple images on AddFragment 

### PR DESCRIPTION
**Pull request by paul-ndl**
We can now select multiples images when we create an apartment and these pictures are not uploaded on Firebase until we press the `Terminate` button.

### CODE REVIEW

File: app/src/androidTest/java/ch/epfl/sweng/hostme/AddFragmentTest.java [hunk 1]
```diff
@@ -61,6 +62,10 @@ public static void setUp() {
         FirebaseApp.initializeApp(ApplicationProvider.getApplicationContext());
     }
 
+    @Rule
+    public GrantPermissionRule accessRule = GrantPermissionRule.grant(android.Manifest.permission.CAMERA, Manifest.permission.WRITE_EXTERNAL_STORAGE, Manifest.permission.INTERNET, Manifest.permission.READ_EXTERNAL_STORAGE);
```
Comment (Scott2Williams) [inline app/src/androidTest/java/ch/epfl/sweng/hostme/AddFragmentTest.java:66]: using full path to `CAMERA` permission when Manifest is imported

**Review by natgate — APPROVED** [review]
Nice feature! Well done

**Review by Scott2Williams — APPROVED** [review]
We may get some comments from the coaches about the stability of maintaining a static array of URI content, but since it's all retrieved during a single external intent, I don't imagine we'll have any problems. Nice work !

File: app/src/androidTest/java/ch/epfl/sweng/hostme/AddFragmentTest.java (hunk 1 above)
Comment (paul-ndl) [reply app/src/androidTest/java/ch/epfl/sweng/hostme/AddFragmentTest.java:66]: I tried to add permission that are working on AccountFragment tests but it doesn't seem to work...

**Pull request merged**